sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDAFSuite.scala

```diff
@@ -149,6 +149,14 @@ class HiveUDAFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
       }
     }
   }
+
+  test("HiveUDAF with 0 rows throws NPE") {
+    sql("create table abc(a int)")
+    sql("select histogram_numeric(a,2) from abc").show
```

Comment:
just doing `.show` checks that there is no Exception, but doesn't actually check the result.  shouldn't these instead be calls to `checkAnswer`?

Reply:
Agreed. Updated the testcases to do ``checkAnswer``